File: 04_08_ternary_operator.js
let n_cats = 10

if (n_cats < 3) {
    console.log("So sad")
}
else if (n_cats >= 3 && n_cats <= 5) {
    console.log("Nice!")
}
else {
    console.log("You are a lucky person!")
}

// You are a lucky person!

let n_cats = 3
let message = n_cats >= 3 ? "Nice!" : "So sad"
console.log(message)
// Nice!

File: 04_08_ternary_operator.py
n_cats = 10

if n_cats < 3:
    print("So sad")
elif n_cats >= 3 and n_cats <= 5:
    print("Nice!")
else:
    print("You are a lucky person!")

# You are a lucky person!

n_cats = 3
message = "Nice!" if n_cats >= 3 else "So sad"

print(message)
# Nice!